duckduckgo/macos-browser
# Daily os version counter pixel added

Task/Issue URL: https://app.asana.com/0/1177771139624306/1205746017002941/f

**Description**:
m.mac.daily-os-version-counter, that carries the major version of the user's macOS (e.g. 10, 11, 12, 13).

**Steps to test this PR**:
1. Run the app
2. Enable Pixel logging (Debug -> Logging -> Pixel)
4. Visit a random website (except duckduckgo.com)
5. Make sure `m.mac.daily-os-version-counter` was NOT fired
6. Open a new tab and search for something
7. Verify `m.mac.daily-os-version-counter` was fired with major os version as a parameter
8. Open another tab and search for something again
9. Verify `m.mac.daily-os-version-counter` was NOT fired again (It should trigger just once a day)
 


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
